tests/jobs/test_job_decorator.py

```diff
@@ -205,30 +230,6 @@ async def execute(self):
     coresys.core.state = CoreState.FREEZE
     assert not await test.execute()
 
-
-async def test_ignore_conditions(coresys: CoreSys):
```

Comment:
The test for each job decorator also confirms that it can be ignored now. So this test specifically for ignoring conditions is unnecessary.